open_academy/web_services/ws.py

```diff
@@ -0,0 +1,64 @@
+import xmlrpc.client
+
+url = "http://localhost:8069"
+db = "rd-mydb"
+username = "admin"
+password = "admin"
+
+# Authentication
+common = xmlrpc.client.ServerProxy('{}/xmlrpc/2/common'.format(url))
+uid = common.authenticate(db, username, password, {})
+models = xmlrpc.client.ServerProxy('{}/xmlrpc/2/object'.format(url))
+
+# Show sessions and seats
+
+session_info = models.execute_kw(db, uid, password, 'open.academy.session',
+                                 'search_read', [], {'fields': ['name', 'seats']})
+
+for session_and_seats in session_info:
+    print("The session '{}' has {} seats".format(session_and_seats["name"], session_and_seats["seats"]))
+
+print("\n")
+
+# Create a session
+
+courses_info = models.execute_kw(db, uid, password, 'open.academy.course',
+                                 'search_read', [], {'fields': ['name']})
+instructors_info = models.execute_kw(db, uid, password, 'res.partner',
+                                     'search_read', [[['instructor', '=', True]]],
+                                     {'fields': ['name']})
+possible_attendees = models.execute_kw(db, uid, password, 'res.partner',
+                                 'search_read', [], {'fields': ['name']})
+
+print("Possible attendees (ID, name):")
+for attendee in possible_attendees:
+    print(attendee["id"], attendee["name"])
+print("\n")
+print("Available courses:")
+for course in courses_info:
+    print(course["name"])
+print("\n")
+print("Available instructors:")
+for instructor in instructors_info:
+    print(instructor["name"])
+
+# Fill the information
+course_name = "Nikon"
+new_session_name = "Nikon II"
+duration_in_days = 1
+number_of_seats = 5
+instructor_name = "Brandon Freeman"
+
+course_id = (next(filter(lambda d: d.get("name") == course_name, courses_info), None))["id"]
+instructor_id = (next(filter(lambda d: d.get("name") == instructor_name, instructors_info), None))["id"]
+
+id = models.execute_kw(db, uid, password, 'open.academy.session', 'create', [{
+    "name": new_session_name,
+    "course_id": course_id,
+    "duration": duration_in_days,
+    "seats": number_of_seats,
+    "instructor_id": instructor_id,
+    "attendee_ids": [7, 3]
+    }]
+)
```

Comment:
Use the dictionary `session_data`.
```suggestion
id = models.execute_kw(db, uid, password, 'open.academy.session', 'create', session_data)
```